com.microsoft.mrtk.input/Interactors/InteractorVisuals/MRTKRayReticleVisual.cs

```diff
@@ -111,6 +116,32 @@ private void UpdateReticle()
                                 VariableReticle.UpdateVisuals(rayInteractor.isSelectActive ? 1 : 0);
                             }
                         }
+
```

Comment:
We chatted offline...and agreed to 

1. Remove the need for `LocateTargetHoverPoint` and instead have the `SpatialManipulationReticle` query the last item in the `hovered item list`
2. Change `IVariableSelectInteractor.UpdateVisuals` to take a struct that takes in an interactor, hit position, and hit normal.
3. Update existing implementations of `IVariableSelectInteractor` to use new version of `UpdateVisuals`
4. Change `SpatialManipulationReticle` to implement `IVariableSelectInteractor`